text/0110-dynamic-components.md

```diff
@@ -331,7 +337,7 @@ The compiler add to its metadata all of the parsed dynamic imports, its source v
 
 ## Drawbacks
 
-The biggest drawback of this feature is the historic abuse of dynamic component creation in the Lightning Platform. We will add guardrails to prevent developers from shooting themselves in the foot.
+The biggest drawback of this feature is the historic abuse of dynamic imports in the Lightning Platform. We will add guardrails to prevent developers from shooting themselves in the foot.
```

Comment:
This is also the case of dynamic component instantiation. Some developers are using this approach while branching in the template (using `if:true`, `if:false`) is preferable.